main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-steputils/v2/stepenv"
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-io/go-utils/v2/pathutil"
+	"github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/step"
+	"github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/xcodebuild"
+)
+
+func main() {
+	os.Exit(run())
+}
+
+func run() int {
+	logger := log.NewLogger()
+	step := createStep(logger)
+
+	config, err := step.ProcessConfig()
+	if err != nil {
+		logger.Errorf(err.Error())
+		return 1
+	}
+
+	exitCode := 0
```

Comment:
Could/should this be moved up so it could be used on [line 27](https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R27) as well, instead of it returning `1` explicitly?

Reply:
Hmm, I'm not 100% sure it leads to a more readable code:

```
	exitCode := 0

	logger := log.NewLogger()
	step := createStep(logger)

	config, err := step.ProcessConfig()
	if err != nil {
		logger.Errorf(err.Error())
		exitCode = 1
		return exitCode
	}

	result, err := step.Run(*config)
	if err != nil {
		logger.TErrorf(err.Error())
		exitCode = 1
	}

	if err = step.ExportOutputs(*result); err != nil {
		logger.Errorf(err.Error())
		exitCode = 1
	}

	return exitCode
```

In the case of `step.ProcessConfig` error, we need to terminate the step run,
while in the case of `step.Run` error, we need to continue and do the `step.ExportOutputs`.

Usually, we explicitly return an integer literal in the `run` func.

Reply:
Make sense, updated in https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/commits/a50ce81231e55173c4c50bf3c6ab9d50c2d0cc1e